usecases/custom_list_usecase.go

```diff
@@ -183,6 +188,180 @@ func (usecase *CustomListUseCase) AddCustomListValue(ctx context.Context,
 	return value, nil
 }
 
+func (usecase *CustomListUseCase) WriteCustomListValuesToCSV(ctx context.Context, customListID string, w io.Writer) (string, error) {
+	customList, err := usecase.GetCustomListById(ctx, customListID)
+	if err != nil {
+		return "", err
+	}
```

Comment:
Since I am using `usecase.GetCustomListById` and not the repository directly, I assume security check should be "handled" inside the usecase, isn't it ?

Reply:
Since I am using usecase.GetCustomListById and not the repository directly, I assume security check should be "handled" inside the usecase, isn't it ?

Reply:
hmm well, it's a good question, I suppose "yes" it does a permission check but in a way that's not directly obvious. 
This is definitely not where our backend codebase is the most consistent, but I'd say let's still add it in duplicate, doesn't cost anything. 
Or just a comment above the GetCustomListById line pointing out that permissions are checked there